apps/lottie-viewer/package.json

```diff
@@ -36,17 +37,17 @@
     "@svgr/rollup": "^8.1.0",
     "@testing-library/react": "14.0.0",
     "@testing-library/user-event": "^14.6.1",
-    "@types/node": "16.18.38",
+    "@types/node": "^20.17.30",
     "@types/prismjs": "^1.26.5",
     "@types/react": "18.3.1",
     "@types/react-dom": "18.3.0",
     "@types/react-syntax-highlighter": "^15.5.13",
-    "@vitejs/plugin-react": "4.2.1",
+    "@vitejs/plugin-react": "^4.4.1",
     "eslint": "^8.57.0",
     "jsdom": "^26.0.0",
     "typescript": "5.1.6",
-    "vite": "4.5.13",
-    "vitest": "0.32.4"
+    "vite": "^6.3.3",
+    "vitest": "^3.1.2"
```

Comment:
I noticed all the vite packages were really outdated so I updated them, this also necessitated using more modern version of node types.